lib/oelite/fetch/git.py

```diff
@@ -267,6 +267,17 @@ def unpack(self, d):
         return True
 
     def mirror(self, mirror=os.getcwd()):
+        if self.is_local:
+            # Don't mirror if the source is local. Concept taken from fetch
+            # Do check revision exists before accepting though
+            print "Warning: Skipping creating mirror for local path %s"%(self.uri)
+            repo = oelite.git.GitRepository(self.repo)
+            if self.has_rev(repo):
```

Comment:
Why not just return True right away in this case?  Why check for availability of revision when we are not going to do anything with it...